src/service/xhr-impl.js

```diff
@@ -195,9 +176,11 @@ export class Xhr {
    *
    * @param {string} input
    * @param {?FetchInitDef=} opt_init
+   * @param {boolean=} opt_skipProcessing Disables any processing of the
+   *     response, including asserting the response is 2XX.
    * @return {!Promise<!JSONType>}
    */
-  fetchJson(input, opt_init) {
+  fetchJson(input, opt_init, opt_skipProcessing) {
```

Comment:
@jridgewell, could we expose a different API for doing `fetchJson` normalization? Maybe a simple function that transforms `init`?

@dvotyenko, I'm a prospective user of this new API, since Fast Fetch has many different error conditions that need to be reported to different parties and so there's a lot of explicit checking to keep them straight. In principle, this could be extracted from the `catch` block, and my current WIP code does that, but this complicates the code and undermines type safety, so it's not ideal.